syncopy/nwanalysis/ST_compRoutines.py

```diff
@@ -182,7 +182,6 @@ class ST_CrossSpectra(ComputationalRoutine):
 
     computeFunction = staticmethod(cross_spectra_cF)
 
-    backends = [csd]
```

Comment:
I suppose these changes to files in the `nwanalysis` folder should not be part of this pull request

Reply:
Well, yes mr. pedantic ;) On the other hand this just deletes a superfluous line of code and does not change any functionality..

Reply:
You’re right of course. I started reading and was briefly confused and then had to leave my desk. I‘ll be happy to (properly) review the code in the coming days

Reply:
Sure, I guess @pantaray does not insist on doing it!

Reply:
Already removed myself from the reviewer list. Please go ahead @joschaschmiedt Thank you!